ptx/sec_numerical_integration.ptx

```diff
@@ -566,7 +624,14 @@
       <caption>Approximating <m>\int_0^1 e^{-x^2}\, dx</m> using 5 trapezoids of equal widths</caption>
         <!-- START figures/fig_num3a.tex -->
       <image xml:id="img_num3a" width="47%">
-        <description></description>
+      <shortdescription>approximating the integral of e^{-x^2} from 0 to 1 using five trapezoids</shortdescription>
```

Comment:
I don't think the screen reader will read `e^{-x^2}` as you intend. Can you describe the curve otherwise?